CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.X.Y
 * Feat: Replaced `punycode` package usage with `url.domainToASCII` (#630).
 * Feat: World of Padman (2007) - Added support (#636)
+* Feat: Satisfactory - Added support (#442) (By @Smidy13)
```

Comment:
Just a nitpick, please use the format of `(By {name} #{pr})`, as in `(By @Smidy13 #442)`.

Reply:
Updated to show PR instead of issue reference.